### PR TITLE
Warn users needing to fwupdmgr activate before uploading signed reports

### PR DESCRIPTION
**Pull request by hughsie**
Fixes https://github.com/fwupd/fwupd/issues/7009

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: src/fu-util.c [hunk 1]
```diff
@@ -1705,6 +1705,19 @@ fu_util_report_history(FuUtilPrivate *priv, gchar **values, GError **error)
...
+
+		/* needs an extra action */
+		if (fwupd_device_has_flag(dev, FWUPD_DEVICE_FLAG_NEEDS_ACTIVATION)) {
+			fu_console_print(
+			    priv->console,
+			    /* TRANSLATORS: %1 is a device name, e.g. "ThinkPad Universal
+			     * ThunderBolt 4 Dock" and %2 is "fwupdmgr activate" */
+			    _("%s is pending activation; use %s to complete the update."),
+			    fwupd_device_get_name(dev),
+			    "fwupdmgr activate");
```
Comment (superm1) [inline src/fu-util.c:1717]: Is it worth using a split up $0 here?

Comment (hughsie) [reply src/fu-util.c:1717]: not sure i follow; you mean make it into two sentences?

Comment (superm1) [reply src/fu-util.c:1717]: I mean for the command used to launch in case of a renamed binary or not part of PATH.

It's probably overkill.

Comment (hughsie) [reply src/fu-util.c:1717]: Nah, it's a good idea! We do it for `g_autofree gchar *cmd = g_strdup_printf("%s sync", g_get_prgname());` already. Give me 1 min.

**Review by superm1 — APPROVED** [review]

**Pull request merged**